src/Icons/tests/Fixtures/templates/template1.html.twig

```diff
@@ -2,8 +2,10 @@
     <li id="first">{{ ux_icon('user', {class: 'h-8 w-8'}) }}</li>
     <li id="second">{{ ux_icon('user', {"aria-label": "AriaLabel"}) }}</li>
     <li id="third">{{ ux_icon('sub:check', {'data-action': 'string "with" quotes'}) }}</li>
+    <li id="fourth">{{ ux_icon('sub:check', { height: 24, width: "24" }) }}</li>
```

Comment:
Is this valid SVG ? oO

Reply:
Ok so @norkunas gave me an example.... `data-id="22"` is indeed perfectly valid (and legit)